app/views/directives/edit-environment-from.html

```diff
@@ -52,7 +52,7 @@
         </div>
       </div>
 
-      <div ng-if="!$ctrl.isReadonlyAny && !entry.isReadonlyValue" class="environment-from-editor-button">
+      <div ng-if="$ctrl.isEnvFromSelectable()" class="environment-from-editor-button">
```

Comment:
Going to look at place the ```isReadonlyAny``` and ```isReadonlyValue``` back in...

Reply:
Separated out the options length check and encased both ```isReadonlyAny``` and ```isReadonlyValue``` within the ```isEnvFromReadonly``` method